redbot/core/core_commands.py

```diff
@@ -3599,8 +3605,9 @@ async def _set_prefix(self, ctx: commands.Context, *prefixes: str):
 
     @_set.command(name="serverprefix", aliases=["serverprefixes"])
     @checks.admin_or_permissions(manage_guild=True)
-    @commands.guild_only()
-    async def _set_serverprefix(self, ctx: commands.Context, *prefixes: str):
+    async def _set_serverprefix(
+        self, ctx: commands.Context, specified_guild: Optional[discord.Guild], *prefixes: str
```

Comment:
```suggestion
        self, ctx: commands.Context, server: Optional[discord.Guild], *prefixes: str
```
Just to be consistent with the user-facing name "server" used in the command name.